src/main/resources/model/instance.json

```diff
@@ -43,6 +43,7 @@
       "type": "object",
       "properties": {
         "value": {
+          "inventorySearchTypes": "title",
```

Comment:
why identifiers start related to the title search? 
title search is finished in https://issues.folio.org/browse/MSEARCH-16

Reply:
Thanks, it should be `keyword` instead.